CHANGELOG.rst

```diff
@@ -196,6 +196,7 @@ Trivial/Internal Changes
 - Corrected spelling of 'plotting' in timeseries method (changed 'ploting' to 'plotting'). (`#3429 <https://github.com/sunpy/sunpy/pull/3429>`__)
 - Switched to "importlib_metadata" to get package version to speed up import of SunPy. (`#3449 <https://github.com/sunpy/sunpy/pull/3449>`__)
 - Fix tests for `sunpy.data.data_manager` and ensure they are correctly executed with pytest. (`#3550 <https://github.com/sunpy/sunpy/pull/3550>`__)
+- Modified the `search_metadata` function inside `sunpy.net.jsoc` to return the Astropy.Table instead of pandas dataframe. (`#3950 <https://github.com/sunpy/sunpy/pull/3950>`__)
```

Comment:
We don't use this changelog directly. You have to add an RST file into the changelogs folder.

Reply:
do I have to remove this change from here?

Reply:
@nabobalis should I remove the change in CHANGELOG and only upload a file in changes.rst? or do we have to keep both?

Reply:
Remove this change and add a changelog file in to the changelog folder.